content/blog/post/20190425-elasticsearch-survival-guide.md

```diff
@@ -338,10 +336,10 @@ on the safe side, just stick to JSON over HTTP.
 [The official
 driver](https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/index.html)
 will be maintained through the lifetime of the project. It will implement
-community approved features from its competitors. Unless something awkwardly
-goes wrong, the official one will just either leave no room of additional value
-to its competitors or they will just get rusted and disappear. That being said,
-the official REST client is a piece of crap for two main reasons:
+community approved features from its competitors. Unless something terrible
+goes wrong, ///the official one will just either leave no room of additional value
```

Comment:
Meaning is somewhat unclear. I read this as: the official client will assimilate the other clients by simply including their features or failing to do so it will be succeeded by one of the competitors and will be ignored from then on. Correct?

Reply:
Is this a better rephrasing: *Unless something terrible goes wrong, the official client will either assimilate the others by simply including their features or the unofficial clients will just get rusted due to not being able to keep up with the rate of the new features introduced to the Elasticsearch and they will disappear.*

Reply:
I'd like to suggest: 

> Unless something goes terribly wrong, the official client will either assimilate the others by simply including their features or the unofficial clients will just rust away due to not being able to keep up with the rate of new features being introduced in Elasticsearch and finally disappear.